### DISCUSSION TITLE
concept art

### DISCUSSION THREAD

**Discussion by allroundgoodegg** [lllyasviel/Fooocus#904] [General]
Apologies for being new to this, bit I'd like to generate art that looks like concept artists created it for movies and video game characters / environments. Tried the "conceptual art" style and it seems to do nothing from my testing. How do I achieve that "broad brush photoshopped" look?

**Comment by allroundgoodegg** [comment]
To be honest, I've tried styles like Line Art, Comic and Watercolor on the same simple prompt and none of them seem to do anything. What am I doing wrong?

**Comment by lllyasviel** [comment]
uncheck existing styles or write style in prompt to balance multiple styles

**Reply by allroundgoodegg** [reply]
Can confirm after much testing: the Fooocus V2 and Fooocus Sharp styles both prevent other styles from happening. Tested with SAI Pixel Art and with Origami (both very easy to see if they're working).

**Comment by allroundgoodegg** [comment]
So if I want to get that rough, painterly concept art style, how do I get that? I'm trying combinations of things, but everything is just too polished, not showing the brush strokes enough. I'm attaching examples of what I mean by "rough painterly concept art".
![painterly concept art 3](https://github.com/lllyasviel/Fooocus/assets/150489684/4260a80b-dde3-4c28-ba63-e3bff41460d0)
![painterly concept art 2](https://github.com/lllyasviel/Fooocus/assets/150489684/d2bd05ca-56ca-4753-8cdc-6b2f3b484dd2)
![painterly concept art 4](https://github.com/lllyasviel/Fooocus/assets/150489684/d86600ff-f755-4547-a985-f42fa6d26ece)

**Reply by AFOLcast** [reply]
![dontdreamit](https://github.com/lllyasviel/Fooocus/assets/142127797/0d25423e-5b1f-4684-8d60-b5523309b7bb)

**Comment by AFOLcast** [comment]
I use an Artfull Lora at .7 and crank guidance up to 8. I'm away from my PC right now, but I can get you the link if you like to the Lora.

**Reply by allroundgoodegg** [reply]
I would appreciate that. Thank you.

**Comment by lllyasviel** [comment]
Hi, some of these looks "rough painterly concept art". what kind of "rough" you need? if u have an example u may try image prompt. or perhaps find a lora, or may be try that "watercolor" style?

**Reply by allroundgoodegg** [reply]
Can input images be used to suggest style as well as a subject? I had no idea!

**Comment by allroundgoodegg** [comment]
It's this kind where you can see the brushstrokes used in Photoshop. Those quick-and-dirty early stage concept images used for environment work a lot.
![1](https://github.com/lllyasviel/Fooocus/assets/150489684/05436b46-0258-46a8-823d-c40391eb54a6)

**Comment by lllyasviel** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/4574f739-68ab-4513-826a-c513be9a6f55)
sketch of landscape, black and white
seed is 3519688696302160143

if this is not what u want then perhaps you need to try loras

**Reply by allroundgoodegg** [reply]
That helps - thank you. Trying to figure how to add loras but I can't find instructions on the front page of how to do that. You can tell I'm new! Downloaded one which may help from civitai. It's a single .pt file that I put in the models>loras folder but it's not showing up in the interface as a lora option.

**Reply by AkoZ** [reply]
oh ? You can put SD1.5 Loras ? or only SDXL loras ? i thought on ly those sdXL Loras can be used.
